packages/flutter/lib/src/cupertino/dialog.dart

```diff
@@ -407,7 +407,9 @@ class _CupertinoAlertDialogState extends State<CupertinoAlertDialog> {
             top: contentSection,
             bottom: Column(
               children: <Widget>[
-                _Divider(dividerColor: dividerColor, hiddenColor: backgroundColor, hidden: false),
+                SizedBox(
+                width: double.infinity,
+                child: _Divider(dividerColor: dividerColor, hiddenColor: backgroundColor, hidden: false)),
```

Comment:
It seems to me we should either fix this in divider directly, or we need to update the _Divider's doc to mention the parent still needs to pass down a reasonable constraint.  I preferred the former, but I am ok with either since this is private class.

Reply:
edit looks like we do mentioned that in doc. so LGTM then